embeddings/transformation/hf_transformation/to_pandas_transformation.py

```diff
@@ -0,0 +1,16 @@
+from typing import Dict
+
+import datasets
+import pandas as pd
+
+from embeddings.transformation.transformation import Transformation
+
+
+class ToPandasHuggingFaceCorpusTransformation(
+    Transformation[datasets.DatasetDict, Dict[str, pd.DataFrame]]
+):
+    def transform(self, data: datasets.DatasetDict) -> Dict[str, pd.DataFrame]:
+        out = {}
+        for subset in data.keys():
+            out[subset] = data[subset].to_pandas()
+        return out
```

Comment:
maybe simplify like:
```suggestion
        return dict(map(lambda split: (split, data[split].to_pandas()), data))
```
or 
```suggestion
        return {split: data[split].to_pandas() for split in data}
```
?